.github/workflows/ci.yml

```
@@ -6,11 +6,21 @@ on:
   push:
+    branches-ignore:
+      - "test-bumpalicious-*"
+    paths-ignore:
+      - "test/**"
+    tags-ignore:
+      - "v99*"
   pull_request:
     branches: [main, v1, v2]
+    paths-ignore:
+      - "test/**"
+    tags-ignore:
```

actionlint (1.7.7)

"tags-ignore" filter is not available for pull_request event. it is only for push event

(events)